packages/flutter_tools/test/general.shard/android/template_test.dart

```diff
@@ -0,0 +1,48 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:file/file.dart';
+import 'package:file/memory.dart';
+import 'package:flutter_tools/src/base/logger.dart';
+import 'package:flutter_tools/src/isolated/mustache_template.dart';
+import 'package:flutter_tools/src/template.dart';
+
+import '../../src/common.dart';
+
+void main() {
+
+  testWithoutContext('kotlin reserved keywords', () {
+    final FileSystem fileSystem = MemoryFileSystem.test();
+    final BufferLogger logger = BufferLogger.test();
+    final Directory rootDir = fileSystem.systemTempDirectory.createTempSync('flutter_template_test.');
```

Comment:
you don't need to create a temp dir if you're using a memory file system, you just use currentDirectory